.github/workflows/ci.yml

```
@@ -20,6 +20,17 @@ jobs:
           --health-timeout 5s
           --health-retries 5
     
+    strategy:
+      fail-fast: false
+      matrix:
+        # Set N number of parallel jobs you want to run tests on.
+        # Use higher number if you have slow tests to split them on more parallel jobs.
+        # Remember to update ci_node_index below to 0..N-1
+        ci_node_total: [4]
+        # set N-1 indexes for parallel jobs
+        # When you run 2 parallel jobs then first job will have index 0, the second job will have index 1 etc
+        ci_node_index: [0, 1, 2, 3]
```

Great that you made it work, but for projects to start we might be good with just
```suggestion
        ci_node_total: [1]
        # set N-1 indexes for parallel jobs
        # When you run 2 parallel jobs then first job will have index 0, the second job will have index 1 etc
        ci_node_index: [0]
```

---

Okay we'll keep in this way to continue with our work/ tests but before merging into master we'll change and also we'll add the documentation explaining how to use